### DISCUSSION TITLE
warning when shulkers are placed outside of claims

### DISCUSSION THREAD

**Discussion by elaineandsparky** [GriefPrevention/GriefPrevention#1046] [Ideas]
As far as I can see, players get warned when they place a chest outside of a claim and also when they place blocks outside of a claim but there is no warning when they place a shulker box.

We had a griefing incident on our server that hapened as a player accidently placed a shulker just outside his claim and I believe a warning could have stoped this incident from happening.

**Comment by bigpresh** [comment]
That would seem sensible, and likely quite easy.

In fact, I wonder if that check should be genericised to placing any inventoried item outside a claim, to also cover e.g. furnaces, smokers etc?

**Comment by RoboMWM** [comment]
There are some that find the warning annoying though, and especially for blocks that are generally placed temporarily (shulkerboxes to access something and then put back) idk if this is ideal. The chest and I think a few other "starter" blocks are set to remind the player as these are items that are easily attained early on, while also generally used for permanent structures.

If verbose warning is desired (which is what this is sounding like) then I'd say it makes more sense for an addon IMO where you can then add any blocks you want, and even more relevant features such as frequency and a way to silence future notifications.
